pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -600,6 +600,7 @@ func (s *Summarizer) runFinalPassOnCheckWrapper(cw *CheckWrapper) {
 	nodesMap := s.getNodesMapOfCheckWrapper(cw)
 	nodeCount := len(nodesMap)
 	logrus.Debugf("id: %s nodeCount: %d", cw.ID, nodeCount)
+	fmt.Println(cw.Result)
```

Comment:
Is this print statement intended or only for testing?

Reply:
This was for testing. updating the PR after removing this.